frontend/src/lib/translations.ts

```diff
@@ -135,6 +135,18 @@ export const translationIds = {
 		editUserQuestionTitle: "Frage an Benutzer editieren",
 		expectedAgeData: "Voraussichtliches Alter Daten",
 		maxFileSizeIs: "Die maximale Dateigröße für ein Bild beträgt",
+		deletionWillAffect: "Durch die Löschung werden diese Elemente gelöscht:",
+		deletion: {
+			affectedQuestionAnswers: "Fragen Antworten",
+			affectedMilestones: "Meilsteinen",
+			affectedAnswers: "Meilsteinen Antworten",
+		},
```

Comment:
the admin translation page assumes these are all `string:string` pairs, i.e. no nested objects, otherwise this breaks the admin translation page (I should have put a type def when defining this object to make this explicit, my bad!)